nengo/__init__.py

```diff
@@ -10,9 +12,30 @@
 """
 
 import logging
+import sys
 
 from .version import version as __version__
 
+if sys.version_info < (3, 4):
+    raise ImportError("""
+You are running Python version %s with Nengo version %s.
+Nengo requires at least Python 3.4.
+
+The fact that this version was installed on your system probably means that you
+are using an older version of pip; you should consider upgrading with
+
+ $ pip install pip setuptools --upgrade
+
+There are two options for getting Nengo working:
+
+- Upgrade to Python >= 3.4
```

Comment:
Note this message and check `(3, 4)` will also need to be updated to 3.5 following #1514.

Reply:
Done